### PR TITLE
Ch all crud

### PR DESCRIPTION
**Pull request by Isaackomeza**
#### What does this PR do?

- Implementing CRUD operation in messages
- Implementing CRUD operation in users
- Implementing CRUD operation in blogs
- Implementing CRUD operation in comments

#### Description of Task to be completed?

- Implementing RESTful API using express node.js

#### How should this be manually tested?

- Testing all CRUD operations in postman

#### Any background context you want to provide?

- Blog owner can create, read, update and delete blog 
- Admin can be able to manage messages, users, and comments

#### What are the relevant pivotal tracker stories?

- https://trello.com/c/mCdRezTC/11-implement-blog-api

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/Q

### CODE REVIEW

File: src/controllers/blogs-controllers.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        blog: 'blog successfully created',
+        data: blog,
+    });
+
+};
+
+export const update=(req,res)=>{
+    const id = req.params.id;
+    const blog = blogs.filter((blog) => {
+        return blog.id === id;
```
Comment (Monehin) [inline src/controllers/blogs-controllers.js]: You should use **Find**  not `filter`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

File: src/controllers/blogs-controllers.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        error: 'blog not found',
+    });
+    
+}
+
+export const deleteBlog=(req,res)=>{
+    const id=req.params.id;
+    const blog = blogs.filter((blog) => {
+        return blog.id === id;
+    });
```
Comment (Monehin) [inline src/controllers/blogs-controllers.js]: user find
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,37 @@
 import express from 'express';
+import {createError} from 'http-errors';
 import Mroute from './routes/messages-routes';
+import Blogroute from './routes/blogs-routes';
+import Userroute from './routes/users-routes';
+import Commentroute from './routes/comments-routes';
```
Comment (Monehin) [inline src/index.js]: Use a standard  **camelCasing** variable naming convention

`userRoute`, `blogRoute` etc

Comment (Isaackomeza) [reply src/index.js]: I think I'm done, you can review it again

**Pull request merged**